code/game/machinery/doors/airlock.dm

```diff
@@ -1349,7 +1349,8 @@ About the new airlock wires panel:
 		else
 			user.visible_message("<span class='notice'>[user] cuts down [note] from [src].</span>", "<span class='notice'>You remove [note] from [src].</span>")
 			playsound(src, 'sound/items/wirecutter.ogg', 50, 1)
-		note.forceMove(get_turf(user))
+		if(!user.put_in_active_hand(note) && !user.put_in_inactive_hand(note))
```

Comment:
```suggestion
		if(!user.put_in_hands(note))
```

Reply:
Good one. Didn't know that existed. Don't need the if check there though